src/HelpersServiceProvider.php

```diff
@@ -66,15 +66,37 @@ protected function extendValidator(): void
             return !$result;
         });
 
-        Validator::extend('list_exists', function ($attribute, $value, $parameters) {
+        Validator::extend('list_exists', function ($attribute, $value, $parameters, $validator) {
             if (count($parameters) < 1) {
+                $validator->errors()->add($attribute, 'You must add at least 1 parameter');
+
                 return false;
             }
 
             $table = Arr::get($parameters, 0);
             $keyField = Arr::get($parameters, 1, 'id');
 
-            if (!empty(Arr::get($parameters, 2))) {
+            $tableFields = DB::getSchemaBuilder()->getColumnListing($table);
```

Comment:
it's not good, to get table data inside the validation rule, let's keep this case without user readable error